app/src/main/java/nu/yona/app/api/manager/network/RestApi.java

```diff
@@ -365,6 +394,17 @@
     @POST
     Call<Void> postMessage(@Url String url, @Header(NetworkConstant.YONA_PASSWORD) String password, @Header(NetworkConstant.ACCEPT_LAUNGUAGE) String acceptLanaguage, @Body MessageBody body);
 
+    /********
+     * ACTIVITY  @param url the url
+     *
+     * @param nextActivityUrl      the url
+     * @param password the password
+     * @return the daily activity
+     */
+    @GET
+    Call<EmbeddedYonaActivity> getNextActivityWithUrl(@Url String nextActivityUrl, @Header(NetworkConstant.YONA_PASSWORD) String password, @Header(NetworkConstant.ACCEPT_LAUNGUAGE) String acceptLanaguage);
```

Comment:
This fetches a page of activities. Whether the first or the next is unknown, so just call it ``getActivities``.